### PR TITLE
Fix deprecated constant MYSQL_ASSOC + session_write notices

### PR DESCRIPTION
**Pull request by Hypolite**

### CODE REVIEW

File: maintenance.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<div class="homepage-wrapper">
+    <h1 class="header">
+        Friendica &nbsp; &nbsp;<br>&nbsp; &nbsp; Directory
+    </h1>
+    <h2>
+        Directory under maintenance
+    </h2>
+
+    <p class="about">
+        The Friendica Directory is unavailable at this time, please connect with <a href="https://friendica.mrpetovan.com/profile/hypolite">hypolite@friendica.mrpetovan.com</a> for updates about its status.
```
Comment (annando) [inline maintenance.html]: I guess your other PRs are based upon this one? There is this issue with the mail address that @tobiasd had addressed.

Comment (Hypolite) [inline maintenance.html]: You're right, I committed directly to my master and then I built the two other issues branches on my master as well, which means that the commits with the maintenance page appears everywhere, both in this PR as well as in the branches' PR.

I'm not sure I can make it a setting as it is supposed to be a static page that should work even if nothing else does.

Comment (annando) [inline maintenance.html]: You could maybe include a simple config file for that purpose?

Comment (tobiasd) [inline maintenance.html]: Does the directory not have a `.htconfig.php` file? I simple assumed so ;-)

Comment (Hypolite) [inline maintenance.html]: I will change it in a further PR.

**Pull request merged**